x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSizeXS * 5;
+    border-right: $euiBorderThin;
```

Comment:
Could I suggest putting this as a `border-left` on `.appSearchResult__toggleExpandButton` instead? That way when we have more than 1 action buttons in a row (e.g.: Curations results, which has both a toggle and a "star" button), each button gets its own border without any extra effort